### DISCUSSION TITLE
How do you install on server?

### DISCUSSION THREAD

**Discussion by CandiceStom** [ImageMagick/ImageMagick#1886] [General]
Pardon my obvious lack of knowledge here.
I've implemented a codepack on my website that requires : 

Server Requirements
PHP 5.x or PHP 7.x
PHP GD or Imagick installed and properly configured
Writing permissions for at least the cache folder

How can I install this online? All the code here seems to only install on a pc. 

Can anyone help?

**Comment by urban-warrior** [comment]
We can only help with the installation of ImageMagick.  Assuming you have a Linux-based host, you simply install it from a repo.  For CentOS, install EPEL then `yum install ImageMagick`.  For Ubuntu, `apt install imagemagick`.  For your other questions, try StackOverflow for help.
